portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/Properties.testcase

```diff
@@ -251,6 +242,53 @@ definition {
 		}
 	}
 
+	@description = "Bug: LRAC-7916 | Automation ID: LRAC-11529 | Test Summary: It is possible to create multiple properties in different workspaces"
+	@priority = "4"
+	test CanCreateSamePropertyNameInDifferentWorkspaces {
```

Comment:
Hey @manuferreira, could you put this test to run in isolation? Tests that create a new workspace need to be run in isolation so as not to interfere with other tests. After this [ticket](https://issues.liferay.com/browse/LRAC-11544?jql=project%20%3D%20LRAC%20AND%20fixVersion%20%3D%20release-3.3.3) we will be able to delete workspace and these cases will not need to be executed in isolation